riscv/src/continuations/bootloader.rs

```diff
@@ -78,22 +78,22 @@ pub fn bootloader_preamble() -> String {
     for (i, reg) in REGISTER_NAMES.iter().enumerate() {
         let reg = reg.strip_prefix("main.").unwrap();
         preamble.push_str(&format!(
-            "    public initial_{reg} = bootloader_input_value({i});\n"
+            "    public initial_{reg} = main_bootloader_inputs.value({i});\n"
```

Comment:
This is a bit unfortunate still! Here the main machine exposes cells in the memory machine as public outputs. Not sure what's a great way to clean this up!

Reply:
OK, this also seems to cause problems with PIL Verifier somehow, it doesn't find the publics! (See failing test) Creating an eStark proof works though!